python/tvm/topi/cumsum.py

```diff
@@ -75,6 +84,12 @@ def maybe_cast(x):
             elif i > axis:
                 axis_mul_after *= value
 
+    if exclusive is None:
+        exclusive = 0
+
+    if reverse is None:
+        reverse = 0
+
```

Comment:
You're not using reverse in the topi?

Reply:
You are right. Sorry, I forgot to remove it.